.github/workflows/unit_mac.yml

```diff
@@ -5,7 +5,7 @@ on:
 
 jobs:
   unit-mac:
-    runs-on: macos-11
+    runs-on: macos-13
```

Comment:
Is there any reason we couldn't use `macos-latest` ?

Reply:
Discussed in Slack, but my thought here is to avoid it changing from under us.